projects/storefrontstyles/scss/theme/high-contrast/dark/_variables.scss

```diff
@@ -0,0 +1,14 @@
+.high-contrast-dark {
```

Comment:
Names of those classes in the global namespace are IMHO too general. Can we make them more specific, e.g. 
`cx-high-contrast-dark`
`cx-theme-high-contrast-dark`
?

Reply:
I agree, let's use the `cx-theme-` prefix

Reply:
done